utils/data_helpers.py

```diff
@@ -270,7 +316,7 @@ def build_encoding_map(column):
         encoding_map[unique_value] = idx
     return encoding_map
 
-def encode_tabular_data(tab_xray_df):
+def encode_tabular_data(tab_xray_df, image_path_prefix:str):
```

Comment:
why the random type definition? We should be consistent with all fxns then

Reply:
Haha yeah, I got confused when switching between projects :D